code/modules/cargo/packs/misc.dm

```diff
@@ -9,20 +9,16 @@
 /datum/supply_pack/misc
 	group = "Miscellaneous Supplies"
 
+/datum/supply_pack/misc/randomised/dirtymags/fill(obj/structure/closet/crate/C)
```

Comment:
I mean, I don't know how code works, I did that because I was copy/pasting from the other tabs. I suppose I can drop the randomised and stick this underneath the dirtymags box?